torch_geometric/explain/explanation.py

```diff
@@ -176,6 +177,48 @@ def visualize_feature_importance(
 
         return ax
 
+    def visualize_subgraph(self, node_index: int, edge_index: Tensor):
```

Comment:
No need to pass `edge_index` here - it is an attribute of `Explanation` - just access it via `self.edge_index`.